packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.html

```diff
@@ -1,19 +1,8 @@
 <div class="nui-step-header-ripple"></div>
-
-<div class="nui-step-icon-state-{{state}} nui-wizard-step-icon" [class.nui-step-icon-selected]="selected">
-    <div class="nui-step-icon-content">
-        <ng-container [ngSwitch]="state">
-            <span *ngSwitchCase="'number'">{{_getDefaultTextForState(state)}}</span>
-            <span class="nui-icon-container">
-                <nui-icon *ngSwitchDefault icon="{{_getDefaultIconForState(state)}}"></nui-icon>
-            </span>
-        </ng-container>
-    </div>
-</div>
-<div class="nui-step-label"
-     [class.nui-step-label-active]="active"
-     [class.nui-step-label-selected]="selected"
-     [class.nui-step-label-error]="state == 'error'">
+<div class="nui-wizard-step-label"
+     [class.nui-wizard-step-label-active]="completed"
+     [class.nui-wizard-step-label-selected]="selected"
+     [class.nui-wizard-step-label-error]="state == 'error'">
```

Comment:
we should use BEM here. e.g.

```suggestion
     [class.nui-wizard-step-header__step-label--active]="completed"
     [class.nui-wizard-step-header__step-label--selected]="selected"
     [class.nui-wizard-step-header__step-label--error]="state == 'error'">
```